include/flecs/cpp/system.hpp

```diff
@@ -297,10 +302,10 @@ class system final : public entity {
             is_each = false;
         }
 
-        std::string signature = build_signature(is_each);
+        flecs::string signature = build_signature(is_each);
 
         if (!signature.length()) {
-            signature = "0";
+            signature = flecs::string_view("0");
```

Comment:
flecs::string assignment operator for (const char*) might be handy not to do such boilerplate

Reply:
I actually explicitly did not want this. What happens here is I'm assigning a `string_view` to a `string` instance, which causes the `string` to hold a non-owning pointer. No copy is created.

The string classes I've built are for internal usage only, and I don't want the copy-by-default behavior of std::string (it's wasteful for most functions). That means the resulting code will be a bit more verbose, but it is unambiguous what is happening from looking at the code.

Reply:
Oh wow, that is smart! Nice